src/commands/app/undeploy.js

```diff
@@ -37,15 +37,24 @@ class Undeploy extends BaseCommand {
     }
     try {
       // undeploy
+
+      try {
+        await runPackageScript('pre-app-undeploy')
+      } catch (err) {
+        this.log(err)
+      }
+
       if (!flags['skip-actions']) {
         if (config.app.hasBackend) {
+          await runPackageScript('undeploy-actions')
```

Comment:
this is incorrect. if there is no undeploy-actions script, run the inbuilt one. see example in `serve-static`: https://github.com/adobe/aio-cli-plugin-app/blob/998c9b37a624ca5a2ee25bf5ee35d7cb03aeec25/src/lib/run-dev.js#L120